### PR TITLE
Make Windows a non-experimental platform in CI

### PR DESCRIPTION
**Pull request by mdboom**
This also fixes various things about the benchmarks so the unit tests pass on Windows.

### CODE REVIEW

File: pyperformance/data-files/benchmarks/bm_asyncio_websockets/run_benchmark.py [hunk 1]
```diff
@@ -23,13 +24,20 @@ async def handler(websocket) -> None:
     stop.set()
 
 
+async def send(ws):
+    try:
+        await ws.send(DATA)
+    except websockets.exceptions.ConnectionClosedOK:
```
Comment (kumaraditya303) [inline pyperformance/data-files/benchmarks/bm_asyncio_websockets/run_benchmark.py:30]: Does it happen on windows only or other platforms too?

**Comment by kumaraditya303** [comment]
There's a failing CI of pypy which seems related to this

**Comment by mdboom** [comment]
> There's a failing CI of pypy which seems related to this

Yeah -- I don't think there is a version of dask that is new enough to be compatible with Windows but old enough to work with PyPy.  I'm trying to figure out how to mark the benchmark as CPython-only, but I'm not sure there is a way.

File: pyperformance/data-files/benchmarks/bm_asyncio_websockets/run_benchmark.py (hunk 1 above)
Comment (mdboom) [reply pyperformance/data-files/benchmarks/bm_asyncio_websockets/run_benchmark.py:30]: Windows only.

**Review by kumaraditya303 — APPROVED** [review]
LGTM

**Pull request merged**